src/app/shared/components/auth-pop-up/auth-pop-up-service/auth-pop-up.service.ts

```diff
@@ -0,0 +1,18 @@
+import { Injectable } from '@angular/core';
+import { Subject } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AuthPopUpService {
+  private readonly _popUpInfo$ = new Subject<string | ''>();
```

Comment:
for what is that ? string | ''
must be enough string

Reply:
oh, yeah
thanks